src/ReactHardwareComponent.js

```diff
@@ -21,14 +20,37 @@ export type ReactHardwareComponentViewConfig = {
 
 export const DEFAULT_VIEW_CONFIG = {
   validAttributes: {
+  },
```

Comment:
I think that the PIN_VIEW_CONFIG should still be the `DEFAULT_VIEW_CONFIG`. I suspect that as we start heading down this path that the default case will still be the same attributes as a `pin`.